docs/design/server_infrastructure.md

```diff
@@ -73,25 +71,64 @@ that with:
 $ source /webapps/{domain}/repo/.venv/bin/postactivate
 ```
 
-## Redis
+### Redis
 
 Redis is installed and served from the application server itself. There is no special
 configuration used for it, and it's not exposed.
 
-## Backup
+### Backup
 
-The database servers are regularly backed-up by AWS.
+The database servers are regularly backed-up by AWS. For more details see
+[database backups](./database_backups.md).
 
 Additional backups are created before starting any deployment with `pg_dump`.
 
-## Deployment
+### Deployment
 
 AMY is deployed with [Ansible](https://docs.ansible.com/) scripts. The standard
 deployment procedure is documented in a file in separate repository containing these
 scripts.
 
+---
+
+## Testing
+
+Testing stage has recently been updated to fix many of the shortcomings listed below in
+[Infrastructure shortcomings and future plans](#infrastructure-shortcomings-and-future-plans).
+
+### AWS services used
+
+* [ECS](https://aws.amazon.com/ecs/) - for running a container with AMY with a load
+  balancer
+* [RDS](https://aws.amazon.com/rds/) - for database (serverless PostgreSQL-flavored Aurora)
+* [CloudFront](https://aws.amazon.com/cloudfront/) is used as CDN for assets
+* [Route 53](https://aws.amazon.com/route53/) for keeping domain records and managing
+  TLS certificates
+* [ECR](https://aws.amazon.com/ecr/) - for storing Docker images
```

Comment:
Could you include what these acronyms stand for? It's tough to keep track of them all!

Reply:
Good point, updated.

Reply:
Much better, thanks!